### PR TITLE
Kanvi/patch files for models

### PR DESCRIPTION
**Pull request by kanvi-nervana**
Add patch files for the models to enable them to use grappler.

### CODE REVIEW

File: test/python/common.py [hunk 1]
```diff
@@ -32,6 +33,13 @@
 class NgraphTest(object):
 
     def with_ngraph(self, l, config=tf.ConfigProto()):
+        rewrite_options = rewriter_config_pb2.RewriterConfig(custom_optimizers=[
```
Comment (sayantan-nervana) [inline test/python/common.py]: in pytests we want to request `ngraph-optimizer` only when we have compiled using grappler right? would this work with the normal build?

File: test/grappler/ssd_mobilenet_v1.patch [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+ 
+ eval_input_reader: {
+   tf_record_input_reader {
+-    input_path: "PATH_TO_BE_CONFIGURED/mscoco_val.record-00000-of-00010"
++    input_path: "/localdisk/dataset/mscoco2014/tf_records_val/coco_val.record"
+   }
+-  label_map_path: "PATH_TO_BE_CONFIGURED/mscoco_label_map.pbtxt"
++  label_map_path: "/localdisk/sairamsu/tf_training/models/research/object_detection/data/mscoco_label_map.pbtxt"
+   shuffle: false
+   num_readers: 1
```
Comment (sharathns93) [inline test/grappler/ssd_mobilenet_v1.patch]: these changes in the config file are not required for grappler and can be removed from the patch.

File: test/python/common.py (hunk 1 above)
Comment (kanvi-nervana) [reply test/python/common.py]: This file got uploaded by mistake. I will remove it.

File: test/grappler/ssd_mobilenet_v1.patch (hunk 2 above)
Comment (kanvi-nervana) [reply test/grappler/ssd_mobilenet_v1.patch]: Not sure how these got added, will remove them.

File: test/grappler/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+|DRAW |draw_inf_ng_optimizer.patch|
+|Inception-resnet-v2 |eval_image_classifier_ng_optimizer.patch|
+|mobilenet-v1 |eval_image_classifier_ng_optimizer.patch|
+|ssd-vgg16 |inference_model_ng_optimizer.patch|
+|UNET |unet_ng_optimizer.patch|
+|Wavenet |fastgen_ng_optimizer.patch|
+|YOLO |build_ng_optimizer.patch|
+|NCF |ncf_main_ng_optimizer.patch|
+|Wide and Deep |wide_deep_ng_optimizer.patch|
+|GNMT|misc_utils_ng_optimizer.patch| Does not run currently because of the Select OP issue|
```
Comment (sayantan-nervana) [inline test/grappler/README.md]: I think you mentioned GNMT runs now @sindhu-nervana ?

Comment (sindhu-nervana) [reply test/grappler/README.md]: yup, it runs!

Comment (sindhu-nervana) [inline test/grappler/README.md]:
```suggestion
|GNMT|misc_utils_ng_optimizer.patch| 
```

**Pull request merged**